custom/auth/ante/ante.go

```diff
@@ -48,6 +49,10 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "sign mode handler is required for ante builder")
 	}
 
+	if options.GovKeeper == nil {
```

Comment:
Maybe we should add similar checks for the DistributionKeeper &  IBCChannelKeeper?

Reply:
this does not belong to the scope of this PR... I will make another PR for this. Can be resolved?